idseq_dag/steps/run_alignment_remotely.py

```diff
@@ -297,6 +297,7 @@ def interpret_min_column_number_string(min_column_number_string,
                                                         max_concurrent, chunk_id)
                 log.write("starting alignment for chunk %s on %s server %s" %
                              (chunk_id, service, instance_ip))
+                instance_iD, job_tag = server.register_job_tag(instance_ip)
```

Comment:
Is this synchronous?  I am wary of increasing ever more the time between finding an instance to run and actually dispatching the SSH job.   Let's make this asynchronous -- start a thread that does this, then join it after the ssh job.

Reply:
Consider using `ThreadWithResult` from `util/thread_with_result.py`.